### PR TITLE
KIALI-1071 Remember state of left-nav menu

### PR DESCRIPTION
**Pull request by aljesusg**
Now we are storing user preferences in redux-store-persist like the collapse of the left menu.

Fix issue https://github.com/kiali/kiali/issues/325

collapse_menu

### CODE REVIEW

File: src/components/Nav/Navigation.tsx [hunk 1]
```diff
@@ -37,7 +37,9 @@ const istioConfigRx = /\/namespaces\/(.*)\/istio\/(.*)/g;
 type PropsType = {
   location: any;
   authenticated: boolean;
+  navCollapsed: boolean;
```
Comment (josejulio) [inline src/components/Nav/Navigation.tsx]: Please remove it (navCollapsed) from the [StateType](https://github.com/kiali/kiali-ui/pull/497/files#diff-a642fcb9370181978f2dc88c65a5e69eR47) if no longer used.

Comment (aljesusg) [reply src/components/Nav/Navigation.tsx]: I forgot it !!! Great thanks @josejulio  changed !!!

**Comment by lucasponce** [comment]
Yes, @josejulio is right I have this
```
Failed to compile.

/home/lponce/git/kiali-ui/src/components/Nav/Navigation.tsx
(95,7): Type '{ selectedItem: string; navCollapsed: boolean; }' is not assignable to type 'Readonly<StateType>'.
  Object literal may only specify known properties, and 'navCollapsed' does not exist in type 'Readonly<StateType>'.
```
Trying to test this PR locally.

**Comment by aljesusg** [comment]
Mmm It was after remove the prop of state, I'll test this.

**Comment by aljesusg** [comment]
Fixed  @lucasponce I forgot remove the state of the constructor. Thanks.

**Comment by aljesusg** [comment]
@josejulio can you review it? I did the changes

**Comment by aljesusg** [comment]
Thanks @josejulio

**Pull request merged**